hub/views.py

```diff
@@ -528,4 +529,28 @@ def get_context_data(self, **kwargs):  # pragma: no cover
 
 
 class StatusView(TemplateView):
-    template_name = "hub/status.html"
+    def get_db_status(self):
+        try:
+            with connection.cursor() as cursor:
+                cursor.execute("select 1")
+                if cursor.fetchone()[0] == 1:
+                    return "ok"
+        except:
+            pass
+        return "error"
+
+    def render_to_response(self, context, **response_kwargs):
+        db_status = self.get_db_status()
+
+        http_status_code = 200
+        if db_status == "error":
+            http_status_code = 500
+
+        return JsonResponse(
+            {
+                "database": db_status,
+                "areas": Area.objects.count(),
```

Comment:
if the database is down this fails and you get an error page so probably only want to run these if the database is up.

Reply:
Thanks. After your comment I realised I could easily test this out by killing the `db` docker container. So I’ve done that, and simplified the `StatusView` code too.